### PR TITLE
Changed array to HashMap 

### PR DESCRIPTION
**Pull request by maksim-khiuttiulia**
Changed array to HashMap  for better performance

### CODE REVIEW

File: src/main/java/cz/upce/fei/inptp/zz/Main.java [hunk 1]
```diff
@@ -3,17 +3,18 @@
 import cz.upce.fei.inptp.zz.entity.DistanceMatrix;
 import cz.upce.fei.inptp.zz.entity.DistanceMatrixLoader;
 
+import java.io.IOException;
```
Comment (rodi0878) [inline src/main/java/cz/upce/fei/inptp/zz/Main.java]: Unnecessary import statement

File: src/main/java/cz/upce/fei/inptp/zz/entity/DistanceMatrixLoader.java [hunk 2]
```diff
@@ -28,7 +28,7 @@ public DistanceMatrix load() {
         } catch (FileNotFoundException ex) {
         } catch (IOException ex) {
         }
-
```
Comment (rodi0878) [inline src/main/java/cz/upce/fei/inptp/zz/entity/DistanceMatrixLoader.java]: Unrelated code format change

File: src/main/java/cz/upce/fei/inptp/zz/entity/DistanceMatrix.java [hunk 3]
```diff
@@ -1,30 +1,29 @@
 package cz.upce.fei.inptp.zz.entity;
 
+import java.util.HashMap;
+
 public class DistanceMatrix {
 
-    private String[] locations;
+    private HashMap<String, Integer> locations;
```
Comment (rodi0878) [inline src/main/java/cz/upce/fei/inptp/zz/entity/DistanceMatrix.java]: It would be better to use interface.

File: src/main/java/cz/upce/fei/inptp/zz/Main.java [hunk 4]
```diff
@@ -3,17 +3,18 @@
...
  */
 public class Main {
     public static void main(String[] args) {
-        DistanceMatrixLoader dml = new DistanceMatrixLoader("matrix1.csv");
-        DistanceMatrix dm = dml.load();
-        
-        System.out.println("A-C " + dm.getDistanceBetweenLocations("A", "C"));
-        System.out.println("B-E " + dm.getDistanceBetweenLocations("B", "E"));
-        System.out.println("D-D " + dm.getDistanceBetweenLocations("D", "D"));
+            DistanceMatrixLoader dml = new DistanceMatrixLoader("matrix1.csv");
```
Comment (rodi0878) [inline src/main/java/cz/upce/fei/inptp/zz/Main.java]: Unrelated format changes

**Comment by maksim-khiuttiulia** [comment]
Changed HashMap to Map, remarks were corrected

**Comment by rodi0878** [comment]
No, PR still contains unrelated changes.
